src/core/graph.test.js

```diff
@@ -441,14 +444,30 @@ describe("graph", () => {
         const exampleGraph = new ExampleGraph();
         [
           [
-            "inEdges",
+            "neighborhood IN",
             exampleGraph.inEdges,
-            (opts) => exampleGraph.graph.inEdges(exampleGraph.root, opts),
+            (opts: any) => {
+              if (opts == null) {
+                opts = {};
+              }
+              opts["direction"] = "IN";
+              return exampleGraph.graph
+                .neighborhood(exampleGraph.root, opts)
+                .map(({edge}) => edge);
+            },
           ],
           [
-            "outEdges",
+            "neighborhood OUT",
             exampleGraph.outEdges,
-            (opts) => exampleGraph.graph.outEdges(exampleGraph.root, opts),
+            (opts: any) => {
+              if (opts == null) {
+                opts = {};
+              }
+              opts["direction"] = "OUT";
+              return exampleGraph.graph
+                .neighborhood(exampleGraph.root, opts)
```

Comment:
Can replace `opts` with `{...(opts || {}), direction: "OUT"}` to save some lines, reduce mutation, and remove an `any`.